langchain4j-core/src/test/java/dev/langchain4j/store/embedding/EmbeddingStoreIT.java

```diff
@@ -63,6 +63,9 @@ void should_add_embedding_with_segment_with_metadata() {
         assertThat(match.embedded().metadata().getLong("long_minus_1")).isEqualTo(-1L);
         assertThat(match.embedded().metadata().getLong("long_0")).isEqualTo(0L);
         assertThat(match.embedded().metadata().getLong("long_1")).isEqualTo(1L);
+        if (testLong1746714878034235396()) {
```

Comment:
Maybe this change will affect other subclass's IT?

Reply:
For those stores that I can test, tests are green. I can't test only Azure Cosmos and Tablestore, but I don't see why this change should affect them, unless this bug is present there, in which case it is good that it will fail and expose it

Reply:
Exactly...What I mean it's that other embedding stores may also have this bug.

Reply:
Chroma had it and this change exposed it :)